cms/apps/media/models.py

```diff
@@ -76,7 +90,15 @@ def __str__(self):
         return self.title
 
     class Meta:
-        ordering = ("title",)
+        ordering = ['title']
+
+    def save(self, force_insert=False, force_update=False, using=None, update_fields=None):
+        super(File, self).save(force_insert, force_update, using, update_fields)
+
+        self.width = self.get_width()
```

Comment:
I'm wondering if it's worth merging these down into `get_dimensions` so there's only one file open required? Would speed things up a little.